espressif/vscode-esp-idf-extension
# [VSC-1258] feature/walkthrough

## Description

Create VS Code Walkthrough for our extension

JIRA: https://jira.espressif.com:8443/browse/VSC-1258

## Type of change

Please delete options that are not relevant.
- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## Steps to test this pull request

1) Install ESP-IDF VS Code extension
2) After installation, walkthrough menu should appear (after the firs installation this will set "idf.showOnboardingOnInit": false")
3) If user uninstalls the extension and installs it again "Walkthrough" menu should not appear, unless in user's settings "idf.showOnboardingOnInit" is set to "true" or not being present)
4) Walkthrough should be present in the menu walkthrough if user runs the command "Help: Welcome" and the ESP-IDF VS Code extension is installed.

## How has this been tested?

As described above

**Test Configuration**:
* ESP-IDF Version: -
* OS (Windows,Linux and macOS): Windows

## Dependent components impacted by this PR:

## Checklist
- [ ] Verified on all platforms - Windows
